### DISCUSSION TITLE
正则表达式无法抓取字符串的后16位

### DISCUSSION THREAD

**Discussion by Bywalks** [projectdiscovery/nuclei#2177] [Q&A] [answered]
I want to extract the last 16 bits of the string, but the regularization of nucleis not supported

For example:

token=86726ecd5fb476313cb4562183c0da6b

by python,

re.search(r"([a-z0-9]{16}$)",token).group() = 3cb4562183c0da6b

But in nuclei, it is not supported $

So, is there any other way to extract the last 16 bits of a string

**Comment by forgedhallpass** [comment, marked as answer]
You should be able to achieve this using the `group` attribute.

See: 
* https://nuclei.projectdiscovery.io/templating-guide/operators/extractors/#dynamic-extractor
* https://github.com/projectdiscovery/nuclei/blob/master/SYNTAX-REFERENCE.md (search for `group:`)
